app/controllers/membership_applications_controller.rb

```diff
@@ -45,11 +45,7 @@ def update
 
       new_upload_file params['uploaded_file'] if params['uploaded_file']
 
-      if changed_to_accepted?(params['membership_application'])
-        accept_application
-        redirect_to edit_membership_application_url(@membership_application)
-        return
-      end
+      @membership_application.applicant_updated_info! if (current_user.id == @membership_application.user.id)
```

Comment:
Per the meeting we had earlier today (2 January) with Ulrika, we decided that a user must explicitly mark the application as ready for review again; that will change the state to 'Ready for review' (the state formerly known as 'pending').  Just `saving` or even updating information by the user is not enough to change the status.